public/locales/en/translation.json

```
@@ -13,6 +13,8 @@
   },
   "loginPage": {
     "title": "Talawa Admin",
+    "communityLogo": "Community Logo",
```

Biome (2.1.2)

[error] The key communityLogo was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)